python/fledge/services/core/routes.py

```diff
@@ -240,6 +242,9 @@ def setup(app):
     app.router.add_route('PUT', '/fledge/service/{service_name}/ACL', acl_management.attach_acl_to_service)
     app.router.add_route('DELETE', '/fledge/service/{service_name}/ACL', acl_management.detach_acl_from_service)
 
+    # Proxy Admin API setup with regex
+    proxy.admin_api_setup(app)
```

Comment:
We could have kept the same way as others


app.router.add_route(..., proxy.handler)

same way for  core proxy.setup